### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by feelfreelinux** [feelfreelinux/octo4a#38] [Announcements]
First full release of octo4a :)

- Implemented a new alpine-based bootstrap, supports Android 4.3+
- Loads of new camera settings
- Web terminal via ttyd
- Extensions system
- More support for plugins
- Better USB configuration
- Corrupted detection installation
- Polished a lot of features

Direct upgrades from 0.0.5 are not possible, please back up your data.

<hr /><em>This discussion was created from the release <a href='https://github.com/feelfreelinux/octo4a/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by f4axk** [comment]
installation error  1.0.0

💻 No bootstrap detected, proceeding with installation
💻 Downloading bootstrap 0.1.6
💻 Bootstrap extracted, setting it up...
🐸 add-user.sh
🐸 fake_proc_stat
🐸 install-bootstrap.sh
🐸 ioctlHook.so
🐸 root
🐸 root-pre5
🐸 rootfs.tar.xz
🐸 run-bootstrap.sh
🐸 install-bootstrap.sh[6]: ../root/bin/minitar: can't execute: Permission denied
🐸 install-bootstrap.sh[6]: cat: <stdout>: Broken pipe
🐸 install-bootstrap.sh[11]: can't create bootstrap/etc/resolv.conf: No such file or directory
🐸 bootstrap ready, run with run-bootstrap.sh
🐸 add-user.sh[5]: can't create bootstrap/etc/passwd: No such file or directory
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
💻 Bootstrap installation done
💻 Bootstrap installed
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
🐸 run-bootstrap.sh[36]: ./root/bin/proot: can't execute: Permission denied
💻 Dependencies installed

back to 0.0.5  OK

**Reply by feelfreelinux** [reply]
What Android version are you using?

**Reply by f4axk** [reply]
Samsung s5
Android version 6.0.1

**Reply by feelfreelinux** [reply]
Please do a fresh install, it should work now

**Reply by f4axk** [reply]
Samsung s5
Android version 6.0.1
test on samsung s10
android 11
OK

installation ok
thank-you for your prompt response
